Documentation/docs/learn/faq.md

```diff
@@ -0,0 +1,561 @@
+
+   |<------------------------- pixel ------------------------->|
+    ______________ ______________ ______________ ______________
+   |              |              |              |              |
+   |______________|______________|______________|______________|
+    15          12 11           8 7            4 3            0
+```
+
+Since DICOM V3, you **cannot** store any overlay in the unused bits of
+the `Pixel Data` (`0x7fe0,0x0010`), thus it imply that the unused bits are
+**zeros**. In ITK, there is no such thing as 12-bit pixel type, thus
+when instanciating your reader you declare a 16-bit pixel type. This
+buffer is then passed back to GDCM which sees an array of 16bits values
+and then simply used the simpliest way to store this buffer back on
+disk: `Bits Stored = 16` and `High Bit = 15`. In case you have not made any
+change to your pixel buffer, implicitly the first 4 bits will be zero
+again. The only difference being that they are not as clearly declared
+as 12 bits.
+
+### I'm working with DICOM images, where can I find more information?
+
+See [ITK/DICOM](https://itk.org/Wiki/ITK/DICOM).
+
+## Data display
+
+### What imaging software is available for displaying medical image files?
+
+Please see the [third party applications](https://itk.org/Wiki/ITK/Third_Party_Applications) page for
+visualization applications that used ITK to perform image
+reading/writing.
+
+### When I view the output of my program, all I see is a black image. What went wrong?
+
+ITK is capable of reading and writing data in a variety of formats and
+bit depths. However many "consumer" image programs only work with
+8-bit data, so when you try to display a 16-bit data file the image will
+appear black. The solution is to use an image display program that
+correctly handles these types of files. A small selection is listed
+above.
+
+[ImageMagick](https://imagemagick.org/index.php) is a particularly useful set
+of tools for image display, manipulation and conversion.
+
+[itkwidgets](https://itkwidgets.readthedocs.io/) is capable of
+correctly loading and displaying all file types supported by ITK
+(including anisotropic images).
+
+## Registration
+
+### Can ITK be used for multi-modal registration? For example, CT-MRI, or MRI-fMRI?
+
+ITK provides several method for performing multi-modality registration.
+These methods are described in detail in the "Registration" chapter of
+[The ITK Software Guide](https://itk.org/ItkSoftwareGuide.pdf). In
+particular, you will find interesting the section where the mutual
+information metric is described.
+
+Examples on multimodality registration are available in
+[Examples/RegistrationITKv4](https://github.com/InsightSoftwareConsortium/ITK/tree/master/Examples/RegistrationITKv4). A demo application using mutual
+information is available under
+[ITKApps/LandmarkInitializedMutualInformationRegistration](https://github.com/InsightSoftwareConsortium/ITKApps/tree/master/LandmarkInitializedMutualInformationRegistration).
```

Comment:
As per https://github.com/InsightSoftwareConsortium/ITK/pull/4313#discussion_r1390272297, mentions to `ITKApps` should be removed. @thewtex ?

Reply:
Yes, ITKApps is no longer maintained. I do no think we should direct people to it.

Reply:
Done in PR #4364.